Documentation/Contributing/development-flow.md

```diff
@@ -207,6 +207,11 @@ All pull requests must pass all continuous integration (CI) tests before they ca
 automatically run against every pull request. The results of these tests along with code review feedback determine whether
 your request will be merged.
 
+For developers who do not want to rely exclusively on the CI for test results but would like
+to test their changes as much as possible locally before pushing
+(a behavior that is not required but highly encouraged), the rook project provides convenient
```

Comment:
just simplifying a bit, since everything in this repo is implicitly in the rook project
```suggestion
(a behavior that is not required but highly encouraged), the unit tests and various linters can be run locally in a developer environment.
```

Reply:
applied and squashed. Thanks for the suggestion! I thought this wording was awkward.